IntegrationTests/PurchaseTester/PurchaseTester.xcodeproj/xcshareddata/xcschemes/WatchPurchaseTester.xcscheme

```diff
@@ -55,46 +55,33 @@
       debugServiceExtension = "internal"
       allowLocationSimulation = "YES"
       notificationPayloadFile = "WatchPurchaseTester Extension/PushNotificationPayload.apns">
-      <RemoteRunnable
```

Comment:
Are you using Xcode 13.2? I've seen these scheme changes before, sometimes Xcode isn't consistent with the order, but I wanna make sure.

Reply:
Yup, I'm on Xcode 13.2.1 😊

Reply:
Ha how did I miss that came out 🙃